src/NzbDrone.Core/DecisionEngine/QualityUpgradableSpecification.cs

```diff
@@ -6,8 +7,8 @@ namespace NzbDrone.Core.DecisionEngine
 {
     public interface IQualityUpgradableSpecification
```

Comment:
This needs a new name, its now combining quality and language, which might be required, if so it needs a new name, if we can come up with a way to separate these further then even better.